sys/Makefile.dep

```diff
@@ -16,6 +16,10 @@ ifneq (,$(filter arduino_pwm,$(FEATURES_USED)))
   FEATURES_REQUIRED += periph_pwm
 endif
 
+ifneq (,$(filter auto_init_gnrc_netif,$(USEMODULE)))
+  USEMODULE += gnrc_netif_init_devs
+endif
```

Comment:
Why can't this be moved to the new gnrc-specific Makefile? At least a comment should be added to explain it, if there is a reason.

Reply:
If it would be renamed to `auto_init_netif_gnrc` or `gnrc_auto_init_netif` it would work. But with `gnrc` in the middle, GNRC's `Makefile.dep` will not be included (unless by chance another module with a name starting or ending with gnrc is used). I'll add the comment.

Reply:
Or we just add `auto_init%_gnrc%` to the check for GNRC's `Makefile.dep` inclusion ;-).

Reply:
I'm not sure if two wildcards are allowed. I once failed to match something like `%foo%` for some reason, while both `%foo` and `foo%` worked.

Reply:
Ok, then let's keep it as is for now.